xbmc/interfaces/generic/ILanguageInvocationHandler.h

```diff
@@ -29,10 +29,14 @@ class ILanguageInvocationHandler
 
   virtual bool Initialize() { return true; }
   virtual void Process() { }
+  virtual void PulseGlobalEvent() { }
+  virtual void RequestAbort(ILanguageInvoker *invoker) { }
```

Comment:
Why rename the other ones to On* but OnAbortRequested->RequestAbort? It does not actually request abort in any way. It's a simple callback like all the other on*s. I also deprecated this callback earlier btw.